utilities/upgrade-service/install-upgrade.sh

```diff
@@ -105,12 +118,13 @@ QUAY_USERNAME=redhat-appstudio
 # Revision of the kubernetes manifests to be used for the installation
 GIT_REVISION="main"
 
-while getopts ':i:u:r:' OPTION; do
+while getopts ':i:u:r:h:' OPTION; do
   case "$OPTION" in
     i) IMG=${OPTARG};;
     u) QUAY_USERNAME=${OPTARG};;
     r) GIT_REVISION=${OPTARG};;
-    ?) echo "Available flag options are:\n[-i] to provide gitops service image (default: quay.io/${QUAY_USERNAME}/gitops-service:latest)\n[-u] to provide QUAY registry username (default: redhat-appstudio)"; exit;
+    h) print_help; exit;
```

Comment:
having an exit code is preferred `exit 0` to indicate that ther was no error and `exit 1` for scenarios with invalid option.